pkg/snapshotter/snapshotter_test.go

```diff
@@ -338,11 +335,8 @@ func TestDeleteSnapshot(t *testing.T) {
 func TestGetSnapshotStatus(t *testing.T) {
 	defaultID := "testid"
 	size := int64(1000)
-	createTimestamp := ptypes.TimestampNow()
-	createTime, err := ptypes.Timestamp(createTimestamp)
-	if err != nil {
-		t.Fatalf("Failed to convert timestamp to time: %v", err)
-	}
+	createTimestamp := timestamppb.Now()
+	createTime := createTimestamp.AsTime()
```

Comment:
Same as above.

Reply:
Response same as above, AsTime() doesn't return a nillable type or an error to check.